docs/docs/faq.md

```diff
@@ -0,0 +1,60 @@
+# Frequently asked questions
+
+### Why is the `NEBARI_KUBECONFIG` file in `/tmp`?
+
+Nebari regenerates this file on every run. Yes, it will be removed by the operating system during its cleanup process, but running the `qhub deploy` command again as Nebari will update/create a `NEBARI_KUBECONFIG` file for you.
+
+### How are Nebari conda user environments created? Who creates them?
+
+The short answer: there are currently *two* ways of creating environments, as we are in the process of migrating Nebari to Conda-Store, and so which way depends on your use-case.
+
+The longer answer: for global environments, you can specify the environment in `qhub_config.yml` and it will be made available for all users and services (e.g., CDSDashboards). Creating the environments in Conda-Store, by comparison, will provide more granular control over certain settings and permissions.
```

Comment:
```suggestion
The longer answer: for global environments, you can specify the environment in `qhub_config.yml`, and it will be made available for all users and services (e.g., CDSDashboards). By comparison, creating the environments through conda-store will provide more granular control over certain settings and permissions.
```

Also this could be easier to read and parse in two bullets